docs/kb/kb1001-202401upgrade.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: 'Upgrade: 2024.01'
+permalink: /kb/upg-202401
+nav_exclude: true
+tags: ["upgrade"]
+is_kb: true
+---
+
+## On this page
+{: .no_toc .text-delta }
+1. TOC
+{:toc}
+
+# Introduction
+This document covers the upgrade process from 2023.07 up to 2024.01. The upgrade process for this specific release is not a straight forward helm upgrade and does require quite a bit of work to get it into a running state.
+
+# Technical Documentation
+## Prerequisites
+- Access to the helm chart version `2.0.0`. You may need to `helm repo update` to grab the latest.
+- Download the prerequisite zip artifact <a href="../../assets/other/2024.01-upgrade.zip" download>here</a>
+- `kubectl` and `helm`
+
+## Guide
+Included in the zipped artifact are a number of `.yaml` files along with a PowerShell upgrade script. The script is very generic and may not be usable in your environment if your instance of CluedIn is heavily modified. As a result, it should only be used as a reference point rather than something you run. The below is the manual steps.
```

Comment:
```suggestion
Included in the zipped artifact are a number of `.yaml` files along with a PowerShell upgrade script. The script is very generic and may not be usable in your environment if your instance of CluedIn is heavily modified. As a result, it should only be used as a reference point rather than something you run. Manual upgrade steps are provided below.
```